baselines/fedmix/fedmix/client.py

```diff
@@ -0,0 +1,127 @@
+"""..."""
+
+import random
+from collections import OrderedDict
+
+import torch
+import torch.autograd as autograd
+import torch.nn.functional as F
+from flwr.client import NumPyClient
+from hydra.utils import instantiate
+
+
+class FedMixClient(NumPyClient):
+    """..."""
+
+    def __init__(self, net, trainloader, device, num_epochs, learning_rate):
+        """..."""
+        self.net = net
+        self.trainloader = trainloader
+        self.device = device
```

Comment:
```suggestion
        self.device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Reply:
As mentioned earlier, let's make the clients choose the device they should use. If for instance you set `num_gpus` in `client_resources` to `0`, then clients won't have access to a GPU and therefore run with `device='cpu'`

Reply:
nevermind! i just realise  you did this `client_fn`, that's 💯